test/e2e/specs/plans/plans__signup-free.ts

```diff
@@ -0,0 +1,119 @@
+	function () {
+		const username = `e2eflowtestingfree${ DataHelper.getTimestamp() }`;
+		const password = SecretsManager.secrets.passwordForNewTestSignUps;
+		const email = DataHelper.getTestEmailAddress( {
+			inboxId: SecretsManager.secrets.mailosaur.inviteInboxId,
+			prefix: username,
+		} );
+		const blogName = DataHelper.getBlogName();
+
+		let page: Page;
+		let userID: number;
+		let bearerToken: string;
+		let userCreatedFlag = false;
+
+		beforeAll( async () => {
+			page = await browser.newPage();
+		} );
+
+		describe( 'Register as new user', function () {
+			it( 'Navigate to Signup page', async function () {
+				const loginPage = new LoginPage( page );
+				await loginPage.visit();
+				await loginPage.clickSignUp();
+			} );
+
+			it( 'Sign up as new user', async function () {
+				const userSignupPage = new UserSignupPage( page );
+				const details = await userSignupPage.signup( email, username, password );
+				userID = details.ID;
+				bearerToken = details.bearer_token;
+
+				userCreatedFlag = true;
+			} );
+		} );
+
+		describe( 'Onboarding', function () {
+			it( 'Select a free .wordpress.com domain', async function () {
+				const domainSearchComponent = new DomainSearchComponent( page );
+				await domainSearchComponent.search( blogName );
+				await domainSearchComponent.selectDomain( '.wordpress.com' );
+			} );
+
+			it( 'Select WordPress.com Free plan', async function () {
+				const signupPickPlanPage = new SignupPickPlanPage( page );
+				await signupPickPlanPage.selectPlan( 'start with a free site' );
+			} );
+
+			it( 'Skip to dashboard', async function () {
+				const startSiteFlow = new StartSiteFlow( page );
+				await Promise.all( [
+					page.waitForNavigation( { url: /.*\/home\/.*/ } ),
+					startSiteFlow.clickButton( 'Skip to dashboard' ),
+				] );
+			} );
+		} );
+
+		describe( 'Validate WordPress.com Free functionality', function () {
+			let sidebarComponent: SidebarComponent;
+
+			it( 'User is on WordPress.com Free plan', async function () {
+				sidebarComponent = new SidebarComponent( page );
+				const plan = await sidebarComponent.getCurrentPlanName();
+				expect( plan ).toBe( 'Free' );
+			} );
+
+			it( 'Navigate to Upgrades > Plans', async function () {
+				sidebarComponent = new SidebarComponent( page );
+				await sidebarComponent.navigate( 'Upgrade', 'Plans' );
+			} );
+
+			it( 'View plan comparison', async function () {
+				const plansPage = new PlansPage( page, 'current' );
+				await plansPage.showPlanComparison();
+			} );
+		} );
+
+		afterAll( async function () {
+			if ( ! userCreatedFlag ) {
+				return;
+			}
+
+			const restAPIClient = new RestAPIClient(
+				{ username: username, password: password },
+				bearerToken
+			);
+
+			const response = await restAPIClient.closeAccount( userID, username, email );
+
+			if ( response.success !== true ) {
+				console.log( `Failed to delete user ID ${ userID }` );
```

Comment:
Do we want to make this stand out as a warning more obviously? Maybe use `console.warn`, or make the text be a bit louder? Just so we notice it and can follow-up on it?

Reply:
I think that would make sense. We don't want to fail the whole build if this process fails, but you're right that it should stand out some more.